webui/react/src/components/kit/Card.tsx

```diff
@@ -0,0 +1,90 @@
+import { Dropdown, MenuProps, Space } from 'antd';
+import React from 'react';
+
+import { ConditionalWrapper } from 'components/ConditionalWrapper';
+import Link from 'components/Link';
+import Icon from 'shared/components/Icon';
+
+import Button from './Button';
+import css from './Card.module.scss';
+
+interface CardProps {
+  actionMenu?: MenuProps;
+  children?: React.ReactNode;
+  clickable?: boolean;
+  disabled?: boolean;
+  height?: number;
+  href?: string;
+  onClick?: () => void;
+  width?: number;
+}
+
+const stopPropagation = (e: React.MouseEvent): void => e.stopPropagation();
+
+const Card: Card = ({
+  actionMenu,
+  children,
+  clickable,
```

Comment:
Could this be derived from onClick and/or href?

Reply:
Usually yes. This is to account for a specific case where we wrap an `OverviewStats` in a `CheckpointModalTrigger` component. I recognize this as sub-optimal, but it's also the same as what we were already doing, and I didn't want to mess with modals. I can go back in an change it if you'd prefer.

Reply:
I'd prefer we derive it since then you can't make the mistake of not setting it correctly.

Reply:
Alright, I'll go in and extract the work that CheckpointModalTrigger is doing then.